UI/EmuScreen.cpp

```diff
@@ -356,6 +357,11 @@ void EmuScreen::sendMessage(const char *message, const char *value) {
 		} else {
 			gstate_c.skipDrawReason &= ~SKIPDRAW_WINDOW_MINIMIZED;
 		}
+	} else if (!strcmp(message, "chat screen")) {
+		releaseButtons();
+		ChatMenu * ch = new ChatMenu();
```

Comment:
by keep the existing chat menu did you mean,
ChatMenu * ch = new ChatMenu() ; <-- create this object one times on emu screen 
screenManager()->push(ch); <-- and call this line on ctrl + c or press the chat button

or you mean create the class like this 
screenManager()->push(new ChatMenu()); 

because we will not use setPointer() again, so its better to not create chat menu pointer right?